packages/rs-sdk/src/sdk.rs

```diff
@@ -260,7 +287,7 @@ impl Sdk {
             .context_provider()
             .ok_or(drive_proof_verifier::Error::ContextProviderNotSet)?;
 
-        match self.inner {
+        let (object, mtd, proof) = match self.inner {
```

Comment:
Let's don't use shortcuts. It will be confusing for other devs

Reply:
done